bin/setup

```diff
@@ -39,7 +39,4 @@ FileUtils.chdir APP_ROOT do
 
   puts "\n== Removing old logs and tempfiles =="
   system! "bin/rails log:clear tmp:clear"
-
-  puts "\n== Restarting application server =="
-  system! "bin/rails restart"
```

Comment:
I personally don't rely on this restart script. Because this would run on container setups where a server wouldn't be up anyway, I thought it was redundant and slowing down remote build process.